src/doc/src/reference/manifest.md

```diff
@@ -91,7 +91,7 @@ a keyword. [crates.io] imposes even more restrictions, such as:
 
 ### The `version` field
 
-The `version` field is formatted according to the [SemVer] specification:
```

Comment:
Please revert this change

Reply:
In the next paragraph, there is a inline link definition: `[Semantic Versioning](https://semver.org/)`. It is the same as [SemVer]. Why not unify them?

Reply:
imo if we consolidated, it would be to `SemVer`.

Structurally, the way it is written works.  It is describing a specific technical point and the next paragraph goes into greater detail, including in the name of the semantics / spec.

Another way this could be written is on first use of `SemVer` we add `(Semantic Versioning)` after.  However, I think the meaning of `SemVer` is fairly well understood and the link can clarify for anyone who doesn't.  We use `SemVer` throughout the docs and there is no single entry point for adding that parenthetical, so we'd either do it everywhere or nowhere.